28308 Release Ticket Numbering (OT1)

tyndmar opened the pull request:
File changed: oe/release#.p
Modified extra release# check to test against oe-relh versus oe-bolh.
Also, implemented spare-char-1 data value on Protagon (was blank) to prevent duplications

Commit:
28308 Release Ticket Numbering (OT1)
File changed: oe/release#.p
Modified extra release# check to test against oe-relh versus oe-bolh.
Also, implemented spare-char-1 data value on Protagon (was blank) to prevent duplications

brad-advantzware reviewed (changes requested):
Move all "extra" checks intot he getNextRelNo.p for all request types.
Do an additional check to make sure the .r-no and release# assignments in the oe-xxxx tables use the getNextRelNo.p.  When processing an unassembled set, I encountered the .r-no already used error.  Let me know if you can't find that and I will try to create steps to reproduce.

Commit:
28308 Release Ticket Numbering (OT1)
File changed: oe/getNextRRelNo.p
added additional test to ensure seq no "keeps up" with last table release and r-nos

Commit:
28308 Release Ticket Numbering (OT1)
File changed: oe/getNextRelNo.p, oe/Release#.p
removed tests from release#.p; added (c) info block
